### PR TITLE
Added JsUtil and MapPimper

### PR DESCRIPTION
**Pull request by szoio**
- Added some Json stuff
- Added map pimpers

### CODE REVIEW

File: src/main/scala/springnz/util/Pimpers.scala [hunk 1]
```diff
@@ -93,4 +94,27 @@ object Pimpers {
       OffsetDateTime.ofInstant(jInstant, DateTimeUtil.UTCTimeZone)
     }
   }
+
+  implicit class MapOptionPimper[A, B](map: Map[A, Option[B]]) {
+    def removeNones(): Map[A, B] = map.filter { case (_, ob) ⇒ ob.isDefined }.mapValues(_.get)
+  }
+
+  implicit class MapPimper[A, B](map: Map[A, B]) {
+    def mapValuesRemoveNones[C](f: B ⇒ Option[C]): Map[A, C] =
```
Comment (mourednik) [inline src/main/scala/springnz/util/Pimpers.scala:103]: A better name might be `partialMapValues`

**Comment by mourednik** [comment]
Looks good. I added another test case for the json unwrapper. It's good that we have a complex json source to find possible edge cases. But I also think its helpful to see the entire expected output for a smaller json.

File: src/main/scala/springnz/util/Pimpers.scala (hunk 1 above)
Comment (szoio) [inline src/main/scala/springnz/util/Pimpers.scala:103]: I know the name is ugly, but it does say what it does at least.

Comment (mpollmeier) [inline src/main/scala/springnz/util/Pimpers.scala:103]: a common naming scheme is `MapOps`
